examples/basic/cdc.py

```diff
@@ -1,9 +1,8 @@
 # SPDX-License-Identifier: BSD-2-Clause
 
-from torii         import Signal, Module
-from torii.lib.cdc import FFSynchronizer
+from torii         import Module, Signal
 from torii.back    import verilog
-
+from torii.lib.cdc import FFSynchronizer
```

Comment:
If this is being moved out into its own import block (which seems reasonable to do), then surely the same should be applied below in examples/formal?

Reply:
Not sure what you mean, it's the same in the formal examples?

Reply:
Looks like we got confused by the way the diff is written, making it look like there would be a resulting blank line. Disregard then!